cmd/pileup/pileup.go

```diff
@@ -0,0 +1,117 @@
+// Command Group: "SAM Tools"
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/dna"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/sam"
+	"log"
+	"strings"
+)
+
+func usage() {
+	fmt.Print(
+		"pileup - Count bases from sequencing data\n\n" +
+			"Usage:\n" +
+			"  pileup [options] in.bam\n\n" +
```

Comment:
Should we have an output file?  I am guessing this prints to stdout, but I am unsure of the format?  Maybe a quick description or a few example lines of output?

Reply:
Yes, added